src/app/core/inventory/inventory.service.ts

```diff
@@ -16,6 +16,23 @@ export interface InventoryZone {
   label: string;
 }
 
+// filter zone options to the control at an index
+export const filterZones = (allZones: InventoryZone[], selectedFlightZones: InventoryZone[], zoneIndex?: number) => {
```

Comment:
Wait - didn't I already see this method?

Reply:
Yeah, up in `flight-form-control-container.component.ts`.  Should this be in both places, or is one leftover from the refactor?

Reply:
You're not wrong. It's not just because you originally wrote that code either. I meant to move that into one place here and failed to finish that refactoring. Will fix.

Reply:
looked like I was using this function in the component, just hadnt removed it from the class. fixed.